docs/configuration.asciidoc

```diff
@@ -1,19 +1,65 @@
 [[configuration]]
 == Configuration
 
-To adapt the Elastic APM agent to your needs, you can configure it using environment variables.
+To adapt the Elastic APM agent to your needs, you can configure it using different configuration sources, which have different naming conventions for the property key. 
```

Comment:
This sentence is a bit rough. What about something like this:

```suggestion
You can utilize configuration options to adapt the Elastic APM agent to your needs. There are multiple configuration sources, each with different naming conventions for the property key.
```

Reply:
This was literally copy pasted from the java doc 😀😀 But I agree, the suggestion is good.